README.md

```diff
@@ -44,6 +44,14 @@ or install
 $ nix profile install github:stellar/stellar-cli
 ```
 
+Use GitHub Action:
+```
+- uses: stellar/stellar-cli
+  # Optionally specify version of the cli. If not provided, latest available version will be used
+  with:
+    version: 22.4.0
```

Comment:
This example isn't correct. See the description inside the action.yml. The preferred way to install is to pick a matching version, since it is hosted within the same repo. The version field was only added for testing.
```suggestion
- uses: stellar/stellar-cli@v22.4.0
```

Reply:
https://github.com/stellar/stellar-cli/pull/1951
https://github.com/stellar/stellar-docs/pull/1402

Reply:
it's only the preferred way… it's required:

```
the `uses' attribute must be a path, a Docker image, or owner/repo@ref
```